client/src/components/map/layers/networks-markers/index.tsx

```diff
@@ -0,0 +1,208 @@
+import { useState, useMemo, useEffect } from 'react';
+
+import { Marker, useMap } from 'react-map-gl/maplibre';
+import Supercluster from 'supercluster';
+
+import { cn } from '@/lib/classnames';
+
+import { LayerProps } from '@/types/layers';
+
+import { OrganizationProperties, ProjectProperties, useMapNetworks } from '@/hooks/networks';
+
+import NetworksPopup, { PopupAttributes } from '@/components/map/networks-popup';
+
+export type NetworksLayerProps = LayerProps & {
+  beforeId?: string;
+};
+
+const sizes = {
+  '1-10': 'h-[40px] w-[20px]',
+  '10-50': 'h-[60px] w-[40px]',
+  '50-100': 'h-[80px] w-[60px]',
+  '>100': 'h-[100px] w-[80px]',
+};
```

Comment:
I don't think the width should be hard-coded, especially where there are more than 100 organisations/projects. Test with bigger numbers such as 3786.

Reply:
I am not sure what is the problem with this. Lidia specified that there were fixed sizes. 
<img width="154" alt="image" src="https://github.com/Orcasa-Platform/orcasa/assets/9701591/fcd1dc24-55bd-414a-8d04-02331f6569a0">

Reply:
No problem if we can fit bigger numbers 😉.